src/app/lib/utilities/variantHandler/index.js

```diff
@@ -4,22 +4,29 @@ export const servicesWithVariants = {
   // the first element in the array is the default variant
   serbian: ['lat', 'cyr'],
   ukchina: ['simp', 'trad'],
+  uzbek: ['default', 'cyr', 'lat'],
   zhongwen: ['simp', 'trad'],
 };
-
 export const variants = ['simp', 'trad', 'lat', 'cyr', 'default'];
-
 const variantCookieConfig = {
   ukchina: 'chinese',
   zhongwen: 'chinese',
 };
 
+export const articleVariantOverride = ({ service, variant, pageType }) => {
```

Comment:
I feel like this function is a bit 🤢. I've made it super explicit to highlight what it does. Not convinced it should stay this way or stay in the file.